query_optimizer/expressions/AttributeReference.cpp

```diff
@@ -57,6 +60,22 @@ ::quickstep::Scalar *AttributeReference::concretize(
   return new ::quickstep::ScalarAttribute(*found_it->second);
 }
 
+std::size_t AttributeReference::computeHash() const {
+  return std::hash<std::size_t>()(static_cast<std::size_t>(id()));
```

Comment:
FYI, the hash implementation for an integer is to return itself. So, we could just return `id()`.